tests/test_models.py

```diff
@@ -97,6 +97,9 @@ def test_predict(preprocess_data):
 @pytest.mark.parametrize("model_class", MODEL_CLASS)
 @pytest.mark.parametrize('task', APPROVED_PARAMS)
 def test_load_and_save(model_class, task):
+    if task == TASK_REGRESSION and model_class == modelgym.RFModel:
+        print("can't test RandomForestClassifier with TASK_REGRESSION")
```

Comment:
shouldn't the model be responsible of reporting it's capabilities? (at least failing promptly as it gets called in an appropriate way)